platforms/shared/charts/bevel-vault-mgmt/templates/_helpers.tpl

```diff
@@ -8,7 +8,7 @@
 Expand the name of the chart.
 */}}
 {{- define "bevel-vault-mgmt.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- default .Chart.Name .Values.global.nameOverride | trunc 63 | trimSuffix "-" -}}
```

Comment:
This should be just nameOverride as using global will mean you can use only one nameOverride for a deployment. Please revert the corresponding changes